### PR TITLE
Improved leave_request performance, less DB operations necessary

### PR DESCRIPTION
**Pull request by BramVer**

### CODE REVIEW

File: ninetofiver/views.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
 from rest_framework_swagger.renderers import SwaggerUIRenderer
 
 
+
```
Comment (roidelapluie) [inline ninetofiver/views.py]: extra nl

Comment (BramVer) [inline ninetofiver/views.py]: It's a better contrast between imports and functions imo.
I can remove it if you want to, but honestly don't see the point?

Comment (roidelapluie) [inline ninetofiver/views.py]: I think we should simply add pep8 to travis-ci checks :)

Comment (BramVer) [inline ninetofiver/views.py]: Agreed, wasn't aware of pep8 though, removing newline and pushing.

**Pull request merged**